src/internal_telemetry/allocations/mod.rs

```diff
@@ -23,6 +23,9 @@ pub(crate) use self::allocator::{
 const NUM_GROUPS: usize = 128;
 pub static TRACK_ALLOCATIONS: AtomicBool = AtomicBool::new(false);
 
+// Delay in milliseconds.
+pub static REPORT_DELAY: AtomicU64 = AtomicU64::new(5000);
```

Comment:
This should be `REPORTING_INTERVAL_MS`, and the comment tweaked to match (it's an interval, not a delay).